needles/browser_search.json

```diff
@@ -0,0 +1,22 @@
+{
+  "tags": [
+    "browser_search"

```

Comment:
This one will break when we merge Chromium 69 with the new UI style.

Reply:
Yup, that’s noted and can be done as a follow-up.